packages/core/apps/core-apps-server-internal/BUILD.bazel

```diff
@@ -38,8 +31,16 @@ filegroup(
   srcs = SRCS,
 )
 
+ASSETS = glob(["assets/**/*"])
+
+filegroup(
+  name = "assets",
+  srcs = ASSETS,
+)
+
 NPM_MODULE_EXTRA_FILES = [
   "package.json",
+  ":assets"
 ]
```

Comment:
So, what does that do exactly?

But I think I spotted the problem:

This PR moved the assets from `{package}/src/assets` to `{package}/assets`. However you did not change the path in the route registered to serve them:

https://github.com/elastic/kibana/blob/0ba81e12d40561f675b0a11977117b4c165417f6/packages/core/apps/core-apps-server-internal/src/core_app.ts#L205-L206

Changing the `Path.resolve(__dirname, './assets')` to target the correct folder should hopefully do the trick.